neuro_extras/data.py

```diff
@@ -88,9 +88,12 @@ async def _create_copier_container(
 
         cmd = f"neuro-extras data cp {args}"
         image = await _parse_neuro_image(NEURO_EXTRAS_IMAGE)
+        cpu_small = self._client.config.presets["cpu-small"]
         return neuro_api.Container(
             image=image,
-            resources=neuro_api.Resources(cpu=2.0, memory_mb=4096),
+            resources=neuro_api.Resources(
```

Comment:
Sorry for jumping in, but please do not use `neuro.jobs.run` method, please use the new `neuro.jobs.start` one instead.

The old one (`.run`) does not store `preset` on the server-side, and this makes a `neuro job generate-run-command` work badly. Also, it doesn't fit well with credits per presets pricing. We cannot drop the old method at the moment because it is used in many places but please gradually migrate to the new one when possible.

Reply:
@romasku , we would need to add "deprecated" warn to `.run`

Reply:
@romasku thanks for heads up! I updated the code